src/main/resources/com/google/api/codegen/php/standalone_sample.snip

```diff
@@ -0,0 +1,65 @@
+@extends "php/common.snip"
+@extends "php/method_sample.snip"
+
+
+@snippet generate(sampleFile)
+  //// [ This is an auto-generated sample file produced by the gapic-generator. Sample name: "{@sampleFile.className}" ]
+  //// STUB standalone sample "{@sampleFile.className}" /////
+
+  // FIXME: Insert here set-up comments that we never want to display in cloudsite. These are seen by users perusing the samples directly in the repository.
+   
+  // [START full_sample]
+
+  // FIXME: Insert here boilerplate code not directly related to the method call itself.
+       
+  @let apiMethod = sampleFile.libraryMethod
+     @let sample = apiMethod.samples.get(0)
+       //     calling form "{@sample.callingForm.toString}"
+       //     valueSet "{@sample.valueSet.id}" ("{@sample.valueSet.title}")
+       //       description: "{@sample.valueSet.description}"
+       //       {@sample.valueSet.parameters}
+       //     apiMethod "{@apiMethod.name}" of type "{@apiMethod.type}"
+
+       // [START core_sample]
+  
+       // FIXME: Insert here code to prepare the request fields, make the call, process the response.
+       
+       /*
+       {@standaloneSample(apiMethod, sample)}
+       */
+       // [END core_sample]
+  
+       // FIXME: Insert here clean-up code.
+         
+     @end
+  @end
+  // [END full_sample]
+@end
+
+@snippet standaloneSample(apiMethod, sample)
```

Comment:
Please make the inline samples in the other file also key off the `callingForm` like you have here, and add a comment in both places to remind us the structure between the two (not necessarily the methods called) should be the same. See the corresponding Python template for an example.